lib/usd/translators/meshWriter.cpp

```diff
@@ -326,19 +282,18 @@ PxrUsdTranslators_MeshWriter::writeMeshAttrs(
         MFnMesh::MColorRepresentation colorSetRep;
         bool clamped = false;
 
-        if (!_GetMeshColorSetData(
-                finalMesh.object(),
-                MString(colorSetName.c_str()),
-                isDisplayColor,
-                shadersRGBData,
-                shadersAlphaData,
-                shadersAssignmentIndices,
-                &RGBData,
-                &AlphaData,
-                &interpolation,
-                &assignmentIndices,
-                &colorSetRep,
-                &clamped)) {
+        if (!UsdMayaMeshUtil::getMeshColorSetData(finalMesh,
+                                                  MString(colorSetName.c_str()),
+                                                  isDisplayColor,
+                                                  shadersRGBData,
+                                                  shadersAlphaData,
+                                                  shadersAssignmentIndices,
+                                                  &RGBData,
+                                                  &AlphaData,
+                                                  &interpolation,
+                                                  &assignmentIndices,
+                                                  &colorSetRep,
+                                                  &clamped)) {
```

Comment:
clang-format will catch this, but for the purposes of this PR, restrict the change to just the function name, or include the formatting changes in a separate commit. This just makes an already large review even larger unnecessarily.